CIP-0100/README.md

```diff
@@ -0,0 +1,298 @@
+---
+CIP: 100
+Title: Governance Metadata
+Category: Metadata
+Status: Proposed
+Authors:
+    - Pi Lanningham <pi@sundaeswap.finance>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/556
+Created: 2023-04-09
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This Cardano Improvement Proposal (CIP) introduces a standardized and flexible metadata format for governance actions in the Cardano ecosystem. While the ledger does not enforce the structure of metadata, providing a standard for metadata will enable better tooling, improved interoperability, and more consistent display across wallets, blockchain explorers, and other tools. This CIP aims to strike a balance between flexibility and structure to facilitate high-quality tooling, without limiting expressivity with regards to user expression.
```

Comment:
```suggestion
This Cardano Improvement Proposal (CIP) introduces a standardized and flexible metadata format for governance events in the Cardano ecosystem. While the ledger does not enforce the structure of metadata, providing a standard for metadata will enable better tooling, improved interoperability, and more consistent display across wallets, blockchain explorers, and other tools. This CIP aims to strike a balance between flexibility and structure to facilitate high-quality tooling, without limiting expressivity with regards to user expression.
```

Since CIP-1694 defines the term "governance action" I think the term's use here is confusing.